src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/SubtypeTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.model.mapper;
+
+import org.junit.Test;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.ScannableItem;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputDocumentType;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.blob.InputScannableItem;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentSubtype;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentType;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.bulkscanprocessor.helper.EnvelopeCreator.inputScannableItem;
+
+public class SubtypeTest {
```

Comment:
This can be converted to a simple parameterised test suite

Reply:
I find this version much simpler than junit**4** parameterised tests.

Reply:
wouldn't be possible without awesome assertJ soft assertions :)